src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c

```diff
@@ -132,18 +132,49 @@ static void* sGameOverTexs[] = {
     gPauseSave20Tex,     gPauseSave21Tex, gPauseSave22Tex, gPauseSave23Tex, gPauseSave24Tex,
 };
 
-static void* sEquipmentTexs[] =
-    LANGUAGE_ARRAY(sEquipmentJPNTexs, sEquipmentENGTexs, sEquipmentGERTexs, sEquipmentFRATexs);
+#if OOT_NTSC
+#define EQUIPMENT_TEXS(language) ((language) != LANGUAGE_JPN ? sEquipmentENGTexs : sEquipmentJPNTexs)
+#define SELECT_ITEM_TEXS(language) ((language) != LANGUAGE_JPN ? sSelectItemENGTexs : sSelectItemJPNTexs)
+#define MAP_TEXS(language) ((language) != LANGUAGE_JPN ? sMapENGTexs : sMapJPNTexs)
```

Comment:
this isnt due to your changes, just want to take the opportunity that I think "texs" is awkward to read and say. it should prob just say the whole word "textures" lol

Reply:
Hm I was about to change this, but there's actually quite a lot of arrays called `Texs` already. There's no precedent for the macros though, should I change those?

Reply:
no it should probably just be changed all at once, if people agree with my sentiment lol

Reply:
I like Texs

Reply:
I would compromise with `Texes` I guess, but pluralizing a word ending in x with just s is weird to look at (and wrong)
But I also dont think texture is much longer/compromising readability

Reply:
I'm a weirdo who will name a 2d array of an already plural something `thingsss`
as in
```py
thingsss = []
for i in range(5):
  thingss = []
  for j in range(5):
    things = Things()
    thingss.append(things)
  thingsss.append(thingss)

for thingss in thingsss:
  for things in thingss:
    print(things)
```